gwcs/schemas/stsci.edu/gwcs/spherical_cartesian-1.0.0.yaml

```diff
@@ -31,9 +31,14 @@ allOf:
   - $ref:  "tag:stsci.edu:asdf/transform/transform-1.1.0"
```

Comment:
@nden does the addition of a required property imply that we should create a new version of this schema?  The validation will fail for files produced with an older copy of the schema, so it seems to me that it's worth doing.

Reply:
I am confident there were no files produced with these models.

Reply:
Oh, it is `transform`. Leaving this to @nden

Reply:
Ah okay!  I commented on the transform line, but I was referring to the spherical_cartesian-1.0.0.yaml schema itself.  If people aren't using the schema yet, then I don't see a problem.